src/ResourceManager/Websites/Commands.Websites/ChangeLog.md

```diff
@@ -23,7 +23,9 @@
 * Updating to use the latest .NET SDK version (2.0.0)
 * Added two new cmdlets: Get-AzureRmDeletedWebApp and Restore-AzureRmDeletedWebApp
 * New-AzureRmAppServicePlan -HyperV switch is added for create app service plan with windows container
-* New-AzureRmWebApp/ New-AzureRmWebAppSlot/ Set-AzureRmWebApp/ Set-AzureRmWebAppSlot - New parameters (�ContainerRegistryUser string -ContainerRegistryPassword secureString -EnableContainerContinuousDeployment) added for creating and managing windows container app
+* New-AzureRmWebApp/ New-AzureRmWebAppSlot/ Set-AzureRmWebApp/ Set-AzureRmWebAppSlot - New parameters (–ContainerRegistryUser string -ContainerRegistryPassword secureString -EnableContainerContinuousDeployment) added for creating and managing windows container app
+* New Cmdlets Enter-AzureRmWebAppContainerPSSession, New-AzureRmWebAppContainerPSSession for entering and creating remote PSSession in container web app
+* New Cmdlet Get-AzureRMWebAppContainerContinuousDeploymentUr - Gets the Container Continuous Deployment Webhook URL
```

Comment:
Move these added lines up to the `Current Release` header.

Reply:
done